roles/v2/pool/pool-config.toml

```diff
@@ -1,6 +1,6 @@
 listen_address = "0.0.0.0:34254"
-# tp_address = "127.0.0.1:8442"
-tp_address = "75.119.150.111:8442"
+tp_address = "127.0.0.1:8442"
+# tp_address = "75.119.150.111:8442"
```

Comment:
Accidental change?

Reply:
Not accidental, you can run either self hosted template provider, or use hosted one we have, just comment out whichever you don't want to with #.

Reply:
But I guess you're referring to commenting out the hosted pool, I think contributors just keep switching and  commiting changes in their local env, imo ideally it should be pointing to local TP, but currently, it's probably easier to connect to a hosted one

Reply:
I've now just copied it elsewhere and refer to it with `-c ~/.stratum/proxy-config.toml`. That's one way to not accidentally touch it :-)